indigo_app/templates/indigo_app/place/_work_chooser.html

```diff
@@ -0,0 +1,85 @@
+{% load i18n %}
+
+<div class="modal-dialog modal-xl" data-component="WorkChooser">
+  <div class="modal-content">
+    <div class="modal-header">
+      <h5 class="modal-title">{% trans 'Choose a work' %}</h5>
+      <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+    </div>
+
+    <div class="card">
+      <div class="card-header">
+        <form
+          id="work-chooser-filter-form"
+          hx-get="{% url 'place_work_chooser_list' place.place_code %}"
+          hx-trigger="keyup changed from:input delay:0.5s, submit"
+          hx-target=".work-list-table tbody"
+        >
+          <input type="hidden" name="field" value="{{ work_field }}">
+          <input type="hidden" name="submit" value="{{ hx_submit }}">
+          <input type="hidden" name="target" value="{{ hx_target }}">
+          <div class="row">
+
+            <div class="col-3">
+              <select
+                class="form-control" name="country" hx-post="{% url 'place_work_chooser' place.place_code %}"
+                hx-target="closest .modal-dialog"
+                hx-swap="outerHTML"
+              >
+                {% for opt in form.country %}
+                  {{ opt }}
+                {% endfor %}
+              </select>
+            </div>
+
+            <div class="col-3">
+              <select
+                class="form-control" name="locality" hx-post="{% url 'place_work_chooser' place.place_code %}"
+                hx-target="closest .modal-dialog"
+                hx-swap="outerHTML"
+              >
+                {% for opt in form.locality %}
+                  {{ opt }}
+                {% endfor %}
+              </select>
+            </div>
+
+            <div class="col-6">
+              <input type="text" name="q" class="form-control" value="{{ form.q.value|default_if_none:'' }}"
+                     placeholder="{% trans 'Search...'%}" autocomplete="off">
+            </div>
+          </div>
+        </form>
+      </div>
+
+      <form
+        id="work-chooser-form"
```

Comment:
This is the form that gets submitted when the work is chosen, using the url provided by the "caller" and targeting the correct element to replace.